contributors/devel/kubemark-guide.md

```diff
@@ -202,10 +234,21 @@ Because there’s no easy way of mocking other managers (e.g. VolumeManager), th
 are not supported in Kubemark (e.g. we can’t schedule Pods with volumes in them
 yet).
 
-As the time passes more fakes will probably be plugged into HollowNodes, but
-it’s crucial to make it as simple as possible to allow running a big number of
-Hollows on a single core.
-
+We currently plan to extend kubemark along the following directions:
+- Progressively introduce more fake components into HollowNodes, keeping it simple
```

Comment:
We might want to optionally have fluentd as part of the hollow node in the future, though I'm not really sure if it contributes any relevant traffic towards the cluster or if it would even have something to log given that components in the hollow node are mocked out. Also, would we ever be interested in one-off pods like heapster, kube-dns, etc?
Removing this point for now (until some new relevant component for node comes up).